pkg/util/runtime/features.go

```diff
@@ -100,6 +100,42 @@ var (
 	// * change the value in site/content/en/docs/Guides/feature-stages.md.
 	// * Ensure that the features in each file are organized categorically and alphabetically.
 	//
+	// ** Feature Promotion: Alpha to Beta for SDK Functions
+	//	* Move alpha feature to beta from:
+	//		- proto/sdk/alpha/alpha.proto to proto/sdk/beta/beta.proto.
+	//		- sdks/go/alpha.go to sdks/go/beta.go
+	//		- sdks/go/alpha_test.go to sdks/go/beta_test.go
+	//		- sdks/csharp/proto/sdk/alpha/alpha.proto to sdks/csharp/proto/sdk/beta/beta.proto
+	//		- sdks/csharp/sdk/Alpha.cs to sdks/csharp/sdk/Beta.cs
+	//		- sdks/csharp/sdk/IAgonesAlphaSDK.cs to sdks/csharp/sdk/IAgonesBetaSDK.cs
+	//		- sdks/csharp/test/AgonesAlphaSDKClientTests.cs to sdks/csharp/test/AgonesBetaSDKClientTests.cs
+	//		- sdks/rust/proto/sdk/alpha/alpha.proto to sdks/rust/proto/sdk/beta/beta.proto
+	//		- sdks/swagger/alpha.swagger.json to sdks/swagger/beta.swagger.json
```

Comment:
So I'm wondering - rather than list every single file (and as we have more beta stuff in other SDKs) - is it easier/better to be more general and write run `make gen-all-sdk-grpc` to generate the files, and `make run-sdk-conformance-tests` and fix everything that is now broken?

Otherwise, having every file is quite brittle over time as things may change. WDYT?

Reply:
I've removed the list of files and made everything more generic. Please let me know if the changes look good to you.